### PR TITLE
Add asreview export import doc

### PR DESCRIPTION
**Pull request by SagevdBrand**
Left everything as NA for now, but after some more investigation on what type of import can be used, will adapt accordingly.

### CODE REVIEW

**Review by ottomattas — APPROVED** [review]
Looks good!

**Comment by ottomattas** [comment]
Is this branch complete now?

**Comment by SagevdBrand** [comment]
> 
> 
> Is this branch complete now?

I think it is ready to be filled in now :)

File: docs/source/intro/datasets.rst [hunk 1]
```diff
@@ -245,6 +251,61 @@ such as EndNote, Mendeley, Refworks and Zotero. All of these are compatible with
...
+|**Swift Active Screener**| X         |          |
++-------------------------+-----------+----------+
+|**Sysrev**               | X         | X        |
++-------------------------+-----------+----------+
+|**SR accelerator**       | X         | X        |
++-------------------------+-----------+----------+
+
+-  ✅ =  The ASReview export can be imported into the third-party review software.
+-  N/A = This format does not exist.
+-  X = Not supported.
```
Comment (J535D165) [inline docs/source/intro/datasets.rst:307]: Let's remove this.

**Comment by ottomattas** [comment]
We are working on another repository (https://github.com/asreview/citation-file-formatting) to take over the descriptions like this.

**Pull request closed without merging**